Tests/Form/Type/FormatterTypeTest.php

```diff
@@ -296,9 +343,43 @@ public function testBuildViewWithFormatter()
             ),
             'ckeditor_context' => null,
             'ckeditor_basepath' => '',
+            'ckeditor_plugins' => array(),
             'ckeditor_toolbar_icons' => $ckEditorToolBarIcons,
         ));
 
         $this->assertSame($view->vars['format_field_options']['data'], $format);
     }
+
+    public function testBuildViewWithDefaultConfigAndPluginManager()
+    {
+        $pool = $this->getMockBuilder('Sonata\FormatterBundle\Formatter\Pool')->disableOriginalConstructor()->getMock();
+        $translator = $this->getMock('Symfony\Component\Translation\TranslatorInterface');
+        $configManager = $this->getMock('Ivory\CKEditorBundle\Model\ConfigManagerInterface');
+        $pluginManager = $this->getMock('Ivory\CKEditorBundle\Model\PluginManagerInterface');
+
+        $defaultConfig = 'default';
+        $defaultConfigValues = array('toolbar' => array('Button1'));
+        $configManager->expects($this->once())->method('getDefaultConfig')->will($this->returnValue($defaultConfig));
+        $configManager->expects($this->once())->method('hasConfig')->with($defaultConfig)->will($this->returnValue(true));
+        $configManager->expects($this->once())->method('getConfig')->with($defaultConfig)->will($this->returnValue($defaultConfigValues));
+
+        $type = new FormatterType($pool, $translator, $configManager, $pluginManager);
+
+        /** @var \Symfony\Component\Form\FormView $view */
```

Comment:
please use this:

```diff
- /** @var \Symfony\Component\Form\FormView $view */
+ /** @var FormView $view */
```

and use use-stetement therefore.
Keep the old code as it is in this PR please. You can refactor this in a different PR.

Thank you